README.md

```diff
@@ -45,161 +37,60 @@ Therefore, please kindly ensure your project includes the following dependencies
   "react-dom"
 ```
 
-Install the peer dependencies in your project:
+To install the dependencies:
 
-```shell
-npm i @emotion/css @emotion/react @emotion/styled @material-ui/core @material-ui/icons @material-ui/lab react react-dom
 ```
+// with npm
+npm i @emotion/css @emotion/react @emotion/styled @material-ui/core @material-ui/icons @material-ui/lab react react-dom
```

Comment:
I wonder if we should combine installing `czifui` with this section, since a project needs all of them anyway lol